templates/openstackclient/bin/tripleo-export-ceph

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env python3
+
+import json
+import os
+import sys
+import tripleoclient.export
+import yaml
+
+CEPH_USER_FN = 'ceph_client_user.json'
+CEPH_USER_KEY = 'ceph_client_user'
+CEPH_DEPLOY_DIRS = (
+    'ceph-ansible',
+    'ceph-adm'
```

Comment:
`s/ceph-adm/cephadm`

The cephadm subdirectory created by config-download in Wallaby is called "cephadm" not "ceph-adm". I'm not sure where you got "ceph-adm" from. Is that in your environment when director operator is used?

https://github.com/openstack/python-tripleoclient/blob/d027aa968add2a8bf1bf1abafd7cd7010fdf5168/tripleoclient/export.py#L212

Reply:
Just a typo, my current env is 16.2/train so ceph-ansible